Energy_Preserving.jl

```diff
@@ -0,0 +1,500 @@
+# Load the packages we will use.
+# These must first be installed using: import Pkg; Pkg.add("package_name")
+#import Pkg; Pkg.add("BSeries")
+using BSeries
+using Latexify  # Only needed for some pretty-printing cells below using `latexify`
+import SymPy; sp=SymPy;
+using LightGraphs
+using Combinatorics
+using RootedTrees
+using LinearAlgebra
+
+#Parametros iniciales
+#stisfies for order <5, and not for 5.
+A = [ 0 0 0 
```

Comment:
Let's separate the examples from the main code.  These can be moved to another file (and eventually to the "test" directory, and/or the documentation).

Reply:
I see that the examples are gone from this file, but they haven't yet been added in a test or documentation.